lib/upload.js

```diff
@@ -112,15 +268,18 @@ class BaseUpload {
    * @param {object} options Optional options for influencing HTTP requests.
    * @return {Promise} The Promise will be resolved/rejected when the requests finish.
    */
-  static terminate (url, options) {
+  static terminate (...args) {
+    const [url, optionsIn] = args
+
     // Count the number of arguments to see if a callback is being provided as the last
     // argument in the old style required by tus-js-client 1.x, then throw an error if it is.
     // `arguments` is a JavaScript built-in variable that contains all of the function's arguments.
-    if (arguments.length > 1 && typeof arguments[arguments.length - 1] === 'function') {
+    if (arguments.length > 1 && typeof args[arguments.length - 1] === 'function') {
```

Comment:
Could you explain this change? Are we not allowed to access `arguments`, but we can use `arguments.length`?

Reply:
TBH I can't explain that 😅 It seems like [the rule](https://eslint.org/docs/rules/prefer-rest-params) doesn't enforce all uses of arguments, I don't know why. I'll remove the other uses too. Or if you think this rule should be disabled we can do that too.